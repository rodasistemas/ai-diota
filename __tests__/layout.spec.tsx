import renderer from 'react-test-renderer';
import type { Metadata } from "next";
import { Inter } from "next/font/google";

import RootLayout, { metadata } from "@/app/layout";
import { JSX } from 'react';

jest.mock("next");
jest.mock("next/font/google");
jest.mock("@/app/globals.css");

const renderTree = (tree: JSX.Element) => renderer.create(tree);
describe('<RootLayout>', () => {
  it('should render component', () => {
    expect(renderTree(<RootLayout children={undefined}    />).toJSON()).toMatchSnapshot();
  });
  
});


describe('metadata', () => {
  
});