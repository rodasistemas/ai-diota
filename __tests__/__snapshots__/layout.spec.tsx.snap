// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RootLayout> should render component 1`] = `
<html
  lang="en"
>
  <body />
</html>
`;
